### ISSUE TITLE
Error on direct installation in IRIS for Windows (x86-64) 2022.1 version zpm 0.5.1

### ISSUE THREAD

**Issue by SergeyMi37** [intersystems-community/secured-rest-api#5]
zapm:DC-SAMPLE-REST>install dc-sample-rest
 
[DC-SAMPLE-REST|dc-sample-rest] Reload START (C:\InterSystems\IRIS22\mgr\.modules\DC-SAMPLE-REST\dc-sample-rest\1.0.1\)
[DC-SAMPLE-REST|dc-sample-rest] Reload SUCCESS
[dc-sample-rest]        Module object refreshed.
[DC-SAMPLE-REST|dc-sample-rest] Validate START
[DC-SAMPLE-REST|dc-sample-rest] Validate SUCCESS
[DC-SAMPLE-REST|dc-sample-rest] Compile START
[DC-SAMPLE-REST|dc-sample-rest] Compile SUCCESS
[DC-SAMPLE-REST|dc-sample-rest] Activate START
[DC-SAMPLE-REST|dc-sample-rest] Configure START
[dc-sample-rest]        Configure FAILURE
[dc-sample-rest]        Activate FAILURE
ERROR! SQLCODE: -118 Сообщение: Пользователь с именем 'Bill' уже существует

**Comment by evshvarov** [comment]
Installing the second time?

**Comment by SergeyMi37** [comment]
Yes, second. I think the installation should be idempotent. How else to update the version?
